README.md

```diff
@@ -1,73 +1,120 @@
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
-
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
-
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
-
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
-
-Thank you for your interest in Tarantool!
-
-[travis-badge]: https://api.travis-ci.org/tarantool/tarantool.svg?branch=master
-[travis-url]: https://travis-ci.org/tarantool/tarantool
-[gitlab-ci-badge]: https://gitlab.com/tarantool/tarantool/badges/master/pipeline.svg
-[gitlab-ci-url]: https://gitlab.com/tarantool/tarantool/commits/master
-[coverage-badge]: https://coveralls.io/repos/github/tarantool/tarantool/badge.svg?branch=master
-[coverage-url]: https://coveralls.io/github/tarantool/tarantool?branch=master
-[groups-badge]: https://img.shields.io/badge/Google-Groups-orange.svg
-[groups-url]: https://groups.google.com/forum/#!forum/tarantool
-[telegram-badge]: https://img.shields.io/badge/Telegram-join%20chat-blue.svg
-[telegram-url]: http://telegram.me/tarantool
-[slack-badge]: https://img.shields.io/badge/Slack-join%20chat-lightgrey.svg
-[slack-url]: http://slack.tarantool.org/
-[gitter-badge]: https://badges.gitter.im/Join%20Chat.svg
-[gitter-url]: https://gitter.im/tarantool/tarantool
+<p align="left">
+  <a href="https://tarantool.io/en/">
+    <img
+      alt="Tarantool"
+      src="https://i.imgur.com/BUYqImx.png"
+      width="400"
+    />
+  </a>
+</p>
+
+**A feature-rich in-memory database.**  It has the performance of a key-value
+cache with the functionality you would expect from a regular database:
+
+* SQL
+* ACID
+* primary and secondary indexes
+* disk engine
+* stored procedures
+* replication
+* sharding
+* and a wealth of modules and connectors from popular languages
+
+Another Tarantool feature is the ability to write code in Lua, a general purpose
+programming language, right inside the database. You can write custom expiration
+strategies and any business logic, and synchronize data with other data sources.
+
+You can use Tarantool for any KV purposes -- or use it to replace Redis when
+you need secondary indexes, a different structure of storage data, SQL, etc.
+
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
+
+## Getting started
+
+### Get hold of Tarantool
+
+* Try online:
+
+  Visit our playground at https://www.tarantool.io/en/try-dev/
+
+* Use a Docker image:
+
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
+  ```
+
+* Install locally:
+
+  * On Ubuntu/Debian/Centos:
+
+    ```bash
+    curl -L http://tarantool.io/installer.sh | VER=2.2 sudo bash
+    ```
```

Comment:
It is better to give a link to our installation instructions. The way with blindly piping a script to an interpreter maybe is okay for CI or experimenting inside a temporary container, but surely is not universal way we should recommend as the general one. Just give the proper link. We plan to make those instructions better.

BTW, use https where possible.

Reply:
We can make the script more universal: check the user distribution and print an error with a link to the download page.